src/pages/landingpage/landingpage.js

```diff
@@ -0,0 +1,39 @@
+import Navbar from '../../components/navbar/navbar'
+import  Slider  from '../../components/slider/slider'
+import Categories from '../../components/categories/Categories'
+import Footer from '../../components/footer/footer'
+import TopProducts from '../../components/topproductcs/topproducts'
```

Comment:
[Barrel export](https://medium.com/@klauskpm/do-a-barrel-export-aa5b79b76b05) for this will be very helpful

Reply:
got it mahesh .will try to do that .👍.thank you for reviewing